task_netcore/task4/WebApi/task4/task4.WEB/Common/Common.cs

```diff
@@ -0,0 +1,14 @@
+﻿using Microsoft.AspNetCore.Http;
+using System.Linq;
+using System.Security.Claims;
+
+namespace task4.WEB.Common
+{
+    public static class Common
+    {
+        public static string GetUserIdByHttpContext(this HttpContext context)
+        {
+            return context.User.Claims.Where(claim => claim.Type == ClaimTypes.NameIdentifier).FirstOrDefault().Value;
```

Comment:
Use `FirstOrDefault()` with condition from `Where`. Also, it's better to use `?.` due to possible NullReference exception